ntc_templates/templates/alcatel_sros_show_port_description.textfsm

```diff
@@ -0,0 +1,22 @@
+Value PORT_ID (\S+)
+Value DESCRIPTION (.+)
+
+Start
+  ^=+\s*$$ -> PreHeader
+  ^. -> Error
+
+PreHeader
+  ^\s*$$ -> End
+  ^Port\s+Descriptions\s+on\s+Slot\s+\S+\s*$$ 
+  ^=+\s*$$ -> Header
+  ^. -> Error
+
+Header
+  ^Port\s+Id\s+Description\s*$$
+  ^-+\s*$$ -> Port
+  ^. -> Error
+  
+Port
+  ^${PORT_ID}\s+${DESCRIPTION}\s*$$ -> Record
+  ^=+\s*$$ -> PreHeader
+  ^. -> Error
```

Comment:
```suggestion
Start
  ^=+\s*$$
  ^\s*$$
  ^Port\s+Descriptions\s+on\s+Slot\s+\S+\s*$$ 
  ^=+\s*$$
  ^Port\s+Id\s+Description\s*$$
  ^-+\s*$$ -> Port
  ^. -> Error
  
Port
  ^${PORT_ID}\s+${DESCRIPTION}\s*$$ -> Record
  ^=+\s*$$ -> Start
  ^. -> Error
```

Reply:
@bennnnnnnn
First off thank you for considering my feedback.
Secondly when you get to accepting any of my edits, please use the Commit suggestion button so I get "commit credit" for my work. Thank you!